File: School source codes/Lab 7/Squarechar.cs
using System;

class MainClass {
  public static void Main (string[] args) {
    char input;
    int size,counter,longsize;
    longsize = 1;
    Console.Write("What's the character? ");
    input = Convert.ToChar(Console.ReadLine());
    Console.Write("What's the size? ");
    size = Convert.ToInt32(Console.ReadLine());
    Console.WriteLine();
    while (longsize <= size) {
    for (counter = 1; counter <= size; counter++)
      {Console.Write(input);}
    Console.WriteLine();
    longsize++;
    }
  }
}

File: School source codes/Lab 7/Squarechar.rb
longsize = 1
counter = 1
print "What's the character?"
input = gets.chomp
print "What's the size?"
size = gets.chomp.to_i
puts "\n"
while (longsize <= size) 
  while (counter <= size)
    print "#{input}"
    counter += 1
  end
  puts "\n"
  longsize += 1
  counter = 1
end